src/axes.jl

```diff
@@ -898,26 +954,37 @@ function axis_drawing_info_3d(sp, letter)
             end
 
             # add major grid segments
-            add_major_or_minor_segments(ticks[1], ax[:grid], grid_segments, 0.012, ax[:tick_direction] !== :none)
+            add_major_or_minor_segments(
+                ticks[1], ax[:grid], grid_segments, 0.012, ax[:tick_direction] !== :none
+            )
 
             # add minor grid segments
             if ax[:minorticks] ∉ (:none, nothing, false) || ax[:minorgrid]
-                add_major_or_minor_segments(minor_ticks, ax[:minorgrid], minorgrid_segments, 0.006, true)
+                add_major_or_minor_segments(
+                    minor_ticks, ax[:minorgrid], minorgrid_segments, 0.006, true
+                )
             end
         end
     end
 
     return (
-        ticks = ticks,
-        segments = segments,
-        tick_segments = tick_segments,
-        grid_segments = grid_segments,
-        minorgrid_segments = minorgrid_segments,
-        border_segments = border_segments
```

Comment:
I think I'm in general for keeping whitespace around `=`

Reply:
Let me explain my position on this: we are limited by the screen width, and it is thus normal and excepted to try to respect a margin limit, especially for people working on editors which do not support line wrapping. Thus, in order to minimize the need for splitting over multiple lines, we should reduce the number of characters. One way to do that is to remove spaces around `=` sign, especially in function calls, where multiple kwargs induce multiple unnecessary spaces. So my position on this is to remove those spaces.

Reply:
I don't see the benefit to be honest. The formatter will split too long lines into horizontal layout anyways and thus there is text should always fit a reasonable screen. On the other hand removing spaces here makes things hard to read even there is plenty of space left and right.

Reply:
I disagree, especially on named tuples, where the name is identical, I find it easier to read when the `lhs` and the `rhs` are glued to the equal sign. I also find it easier to read for named arguments in methods definitions. But anyway this is very opinionated so I do not really care as long as there is consistency. I've reverted to using `whitespace_in_kwargs = true`.